frontend/src/components/home-page/components/filter-dialog.tsx

```diff
@@ -0,0 +1,38 @@
+import { useState } from 'hooks/hooks';
+import Button from '../../common/button/button';
+import Checkbox from '../../common/checkbox/checkbox';
+import styles from '../styles.module.scss';
```

Comment:
Styles modules should be different files for HomeComponent and FilterDialog components and independent from each other.